### DISCUSSION TITLE
How to run in windows

### DISCUSSION THREAD

**Discussion by ZeroCrossblade** [Girbons/comics-downloader#152] [General]
i was wondering can someone explain how do i run the file in windows

**Comment by Girbons** [comment]
what file version did you download? cli or gui?

**Reply by ZeroCrossblade** [reply]
gui

**Reply by ZeroCrossblade** [reply]
i click on the execute file and nothing happen

**Comment by ZeroCrossblade** [comment]
im trying everything but the file will not run

**Reply by Girbons** [reply]
I will take a look at gui this weekend, in the meantime try to use the cli version

**Reply by ZeroCrossblade** [reply]
thank for replying is there a tutorial on how to install it

**Reply by ZeroCrossblade** [reply]
is there any thing i need to install to run the file

**Reply by Girbons** [reply]
it can be executed from the commandline, there's no need to install anything

**Comment by ZeroCrossblade** [comment]
i download the source code file but there no executable file in the folder

**Reply by Girbons** [reply]
you can find the executable here https://github.com/Girbons/comics-downloader/releases/tag/v0.33.9

**Reply by ZeroCrossblade** [reply]
I can not get it to run when i click the exe file the cmd window will flash on and off

**Reply by Girbons** [reply]
you should open the powershell console where the file is located, then execute the file like in the readme.
cli executable is not supposed to be double clicked

**Comment by ZeroCrossblade** [comment]
thank you for your help i got it working now im new to powershell i was wondering is the gui working

**Reply by Girbons** [reply]
I'm glad it worked! Regarding the UI version I need to find some time to fix it

**Comment by ZeroCrossblade** [comment]
ok thank
